source/fe/fe_system.cc

```diff
@@ -1180,14 +1180,17 @@ FESystem<dim,spacedim>::compute_fill (
           FEValuesData<dim,spacedim> &
           base_data    = fe_data.get_fe_values_data(base_no);
 
-          // Copy quadrature points. These are required for computing
-          // the determinant in the FEPolyTensor class. The
-          // determinant is one ingredient of the Piola
-          // transformation, which is applied to correctly map the RT
-          // space from the reference element to the global coordinate
-          // system.
-          if (cell_similarity != CellSimilarity::translation)
```

Comment:
Does the cell similarity check not work here? Maybe it doesn't matter because the base elements will save computations individually?